core/gethwrappers/generated/ocr2dr_oracle/ocr2dr_oracle.go

```diff
@@ -192,6 +221,28 @@ func (_OCR2DROracle *OCR2DROracleCallerSession) GetDONPublicKey() ([]byte, error
 	return _OCR2DROracle.Contract.GetDONPublicKey(&_OCR2DROracle.CallOpts)
 }
 
+func (_OCR2DROracle *OCR2DROracleCaller) GetRequiredFee(opts *bind.CallOpts, arg0 []byte, arg1 OCR2DRRegistryInterfaceRequestBilling) (*big.Int, error) {
+	var out []interface{}
+	err := _OCR2DROracle.contract.Call(opts, &out, "getRequiredFee", arg0, arg1)
```

Comment:
why doesn't this conform to the Go pattern seen elsewhere which would be

`valueReturnedByFunc, err := _OCR2DROracle.contract.Call(opts, &out, "getRequiredFee", arg0, arg1)`

Reply:
Not sure, this is auto-generated.